src/vs/workbench/browser/parts/titlebar/windowTitle.ts

```diff
@@ -113,22 +117,20 @@ export class WindowTitle extends Disposable {
 			this.activeEditorListeners.add(activeEditor.onDidChangeDirty(() => this.titleUpdater.schedule()));
 			this.activeEditorListeners.add(activeEditor.onDidChangeLabel(() => this.titleUpdater.schedule()));
 		}
+
+		// Apply listeners for tracking focused code editor
 		if (this.titleIncludesFocusedView) {
 			const activeTextEditorControl = this.editorService.activeTextEditorControl;
+			const textEditorControls: ICodeEditor[] = [];
```

Comment:
@meganrogge using an array here so that the logic for installing listeners is not repeated.